drivers/w1/Kconfig

```diff
@@ -40,6 +40,15 @@ config W1_SHELL_BUFFER_SIZE
 	  To be able to write more than 8 bytes make
 	  sure to also increase SHELL_ARGC_MAX accordingly.
 
+config W1_IGNORE_SLAVE_COUNT
```

Comment:
Ignore_slave_count is not really descriptive what it is really doing, and could be understood to have exactly the opposite effect.
How about calling it something like `W1_NET_FORCE_MULTIDROP_ADDRESSING`, highlighting in the description that usually this does not need to be enabled.

Reply:
Good point - I have changed the symbol name and updated the help text accordingly.